packages/docs/src/guide/components/tui-box.md

```diff
@@ -0,0 +1,23 @@
+# `<TuiBox>`
+
+Creates flexbox-based box.
+
+```vue-html
+<Box
+  width="100"
+  height="20"
```

Comment:
I don't remember if it was necessary to pass a number here 🤔 I think we didn't add a cast. If we didn't, it's worth a note